packages/react/src/components/Storage/FileUploader/types.ts

```diff
@@ -14,11 +13,10 @@ export interface UploadDropZoneProps extends DragActionHandlers {
 
 export interface FileUploaderProps {
   acceptedFileTypes: string[];
-  components?: Components;
   hasMultipleFiles?: boolean;
   isPreviewerVisible?: boolean;
   isResumable?: boolean;
-  level: LevelInfo;
+  storageLevel: StorageAccessLevel;
```

Comment:
+1 for naming the type `StorageAccessLevel` that makes more sense, but why name the prop `storageLevel`? It doesn't intuitively resolve to private|public|etc. How about `accessLevel` if we don't want the verbose `storageAccessLevel`.